README.md

```diff
@@ -12,7 +12,7 @@ If you are looking for a particular feature, please create an [issue](https://gi
 
 1. Download & Install [RocketType_vx.x.exe](https://github.com/janmechtel/rockettype/releases)
 2. RocketType will run after installation
-3. To run again: type `RocketType` in Windows Start menu 
```

Comment:
I noticed how sometimes there are spaces at the end of the line. I think this is related to different editors since you are on atom and I'm on VS Code. Could you figure out how we prevent this?

Reply:
My editor is set to auto-remove whitespace at the end of lines. I could disable this setting so that it doesn't keep creating changes to random lines or you could check and see if there is a setting like this on your editor.